### PR TITLE
Add a link for the KB about the setup of APIs

### PR DESCRIPTION
**Pull request by mitogh**
See https://central.tri.be/issues/89761

### CODE REVIEW

File: src/Tribe/Google/Maps_API_Key.php [hunk 1]
```diff
@@ -47,9 +47,14 @@ public function filter_tribe_addons_tab_fields( array $addon_fields ) {
 
 			'gmaps-js-api-info-box' => array(
 				'type' => 'html',
-				'html' => '<p>' . sprintf( __( 'We highly recommend that you specify a valid %s for The Events Calendar to use. Doing this will help prevent problems with maps, especially for sites that receive a lot of traffic.',
-						'the-events-calendar' ), '<a href="https://developers.google.com/maps/documentation/javascript/get-api-key" target="_blank">' . __( 'Google Maps API key',
-							'the-events-calendar' ) . '</a>' ) . '</p>',
+				'html' => '<p>' . sprintf(
+					__(
+						'We highly recommend that you specify a valid %s for The Events Calendar to use. Doing this will help prevent problems with maps, especially for sites that receive a lot of traffic. Visit %s for a detailed explanation in how to do it.',
```
Comment (barryhughes) [inline src/Tribe/Google/Maps_API_Key.php]: Numbered placeholders like `%1$s` are really helpful to give translators max flexibility

Comment (mitogh) [reply src/Tribe/Google/Maps_API_Key.php]: Great idea 👍

**Comment by mitogh** [comment]
@barryhughes would you mind take another look at this one thanks.

**Comment by barryhughes** [comment]
Fantastic :+1:

**Pull request merged**